app/views/landing_page/show.html.erb

```diff
@@ -26,9 +26,15 @@
 
 <main class="landing-page" id="content">
   <% @content_item.blocks.each do |block| %>
-    <%= tag.div class: ["govuk-block__#{block.type}", ("govuk-width-container" unless block.full_width?)] do
-      render_block(block)
-    end %>
+    <%= tag.div class: ["govuk-block govuk-block__#{block.type}", ("govuk-block--background" if block.full_width_background?)] do %>
+      <% if block.full_width? %>
+        <%= render_block(block) %>
+      <% else %>
+        <div class="govuk-width-container">
```

Comment:
We can probably refactor this a little so that the if / else is inline, but that is really really minor.

Reply:
Agreed, I was just writing out to be clear.

Reply:
Also can't think of a shorter way of writing it, if you have a suggestion would be happy to add.

Reply:
Yeah, I can't either. Perhaps one to worry about later?